pyaedt/desktop.py

```diff
@@ -99,18 +96,12 @@ def launch_desktop_on_port():
             return False, port
         time.sleep(1)
         k += 1
-    active_after = grpc_active_sessions(
-        settings.aedt_version, non_graphical=settings.non_graphical, student_version=settings.is_student
-    )
     k = 0
     time.sleep(1)
-    while len(active_after) == len(active):  # pragma: no cover
-        time.sleep(1)
-        active_after = grpc_active_sessions(
-            settings.aedt_version, non_graphical=settings.non_graphical, student_version=settings.is_student
-        )
-        if k > timeout:
-            return False, port
+    if port not in grpc_active_sessions(
+        settings.aedt_version, non_graphical=settings.non_graphical, student_version=settings.is_student
+    ):  # pragma: no cover
+        return False, port
```

Comment:
if `port` is not in `grpc_active_sessions`, why do you return `port`? Maybe return `None` instead?

Reply:
to be coherent with expected format